tedana/workflows/tedana.py

```diff
@@ -223,6 +223,24 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     file. For a complete list of the files generated by this workflow, please
     visit https://tedana.readthedocs.io/en/latest/outputs.html
     """
+    out_dir = op.abspath(out_dir)
+    if not op.isdir(out_dir):
+        os.mkdir(out_dir)
+
+    if debug and not quiet:
+        formatter = logging.Formatter(
+                    '%(asctime)s\t%(name)-12s\t%(levelname)-8s\t%(message)s',
+                    datefmt='%Y-%m-%dT%H:%M:%S')
+        fh = logging.FileHandler(op.join(out_dir, 'runlog.tsv'))
```

Comment:
Looking at the artifacts on circle it looks like the logs are all correctly written to screen, but I don't see any saved out to a tsv ?

Reply:
The tsv file is currently only written out if `debug` is used, and I didn't change either of the tests to run on debug. Would you like me to change the behavior (e.g., generate it always) or the tests (i.e., add `--debug` to one of them)?

Reply:
Let's change one of the tests, if that's ok ! Sorry, I totally misread and thought we had that there already. Thanks :)

Reply:
This looks great ! Final question -- right now we just save it as `runlog.tsv` -- should we put the date and time in the filename, in case the same data is run multiple times ?

Reply:
I really don't know what's best practice for logging. Adding in the date and time makes it slightly harder to grab the relevant file. Although with glob or regular expressions it's not that much harder...

Reply:
How do you feel about the ISO format in the filename?
E.g., `runlog_2018-12-11T11:15:26.228355.tsv`.

Reply:
My only concern with having colons in the filename is that I ran into an issue with some other filenames with colons in them just recently -- tar thinks they're living on another machine and freaks out. Not necessarily a problem, here, but I might prefer converting to dots to be safe... WDYT ?

Reply:
That's fine by me. I'll push that now.

Reply:
Wait a second.... what about the CI? The date time for the expected outputs list won't match the  actual outputs.

Reply:
Separate logs files are a good idea, I think. I'll just revert the test.